File: problem set 1/Mario(less)/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n;
    do
    {
        n = get_int("pleas enter number between 1,8 : ");
        // we cant get a number
    }
    while (n > 8 || n < 1);





    // and we can make a mario with space
    for (int i = 0 ; i < n ; i++)
    {


        for (int b = 0 ; b <= i ; b++)
        {


            for (int c = 1 ; c < n - i ; c++)
            {
                printf(" ");     //here we can make space
            }

            for (int z = 0 ; b <= i ; b++)
            {
                printf("#");    // here we can make #
            }


        }

        printf("\n");


    }




}

File: problem set 6 /mario /mario.py
from cs50 import get_int


n = get_int("pleas enter number between 1,8 : ")
# we cant get a number


while n > 8 or n < 1:  # make sure its 1 - 8
    n = get_int("pleas enter number between 1,8 : ")


for i in range(n):
    if i == 0:
        for w in range(n - (i + 1)):
            print(" ", end="")
        for t in range(1):
            print("#", end="")
    else:
        for q in range(n - (i + 1)):
            print(" ", end="")  # make space
        for z in range(i + 1):
            print("#", end="")  # make #
    print()
